styles/style.css

```diff
@@ -0,0 +1,587 @@
+  font-weight: 600;
+  letter-spacing: 0.05em;
+  line-height: 1.5rem;
+  margin-left: auto;
+  margin-right: auto;
+  text-transform: uppercase;
+  width: 330px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgb(50, 104, 145);
+  border-image: initial;
+  border-radius: 3px;
+  padding: 5px 0px;
+  transition: background-color 0.6s ease 0s;
+}
+
+/* main end */
+
+/* footer */
+
+/* email-input */
+
+.signup {
+    width: 1200px;
+    margin: auto;
+    display: flex;
+    border-top: 1px solid black;
+    border-bottom: 1px solid black;
+    padding-bottom: 3rem;
+}
+
+.input-cont {
+    width: 50%;
+}
+
+.input-box {
+    flex: 1;
+    width: 100%;
+    display: flex;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+}
+
+.input-box input {
+    width: 60%;
+}
+
+.input-box>button {
+    width: 40%;
+}
+
+.btn-comment {
+    background-color: #567b95;
+    border: 1px solid #3f5c70;
+    border-radius: 3px;
+    font-size: 11px;
+    font-weight: 700;
+    padding: 11px 12px 8px 12px;
+    border-radius: 0 3px 3px 0;
+}
+
+.text-cont {
+    width: 50%;
+}
+
+.input-cont span{
+  font-size: 90%;
+}
+
+.input-cont ul {
+  display: flex;
+  padding: 0;
+}
+
+.input-cont li {
+  font-size: 60%;
+  color: blue;
+}
+
+.input-cont li:nth-child(1) {
+  padding-right: 10px;
+  border-right: 1px solid black;
+}
+
+.input-cont li:nth-child(2) {
+  padding-left: 10px;
+}
+
+
+
+
+
+/* email-input end */
+
+/* footer  */
+
+.news {
+  width: 1200px;
+  margin: auto;
+  margin-top: 1rem;
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+
+}
+
+.news-items {
+  width: 500px;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  padding-right: 10px;
+  border-bottom: 1px solid black;
+  margin-bottom: 20px;
+}
+
+.grid-item {
+  width: 200px;
+  margin-right: 20px;
+}
+
+.grid-item img {
+  width: inherit;
+
+}
+.grid-item a {
+  width: inherit;
+}
+
+.new-headings {
+  width: 100px;
+
+}
+
+.new-headings a h2 {
+  margin-top: -20px;
+  margin-bottom: 10px;
+}
+
+.new-headings ul li h3 {
+  margin-left: 10px;
+}
+
+.new-headings ul li {
+  border-top: 1px solid rgb(228,228,228);
+}
+.grid-news-items {
+  width: auto;
+
+}
+
+.wraper {
+  display: flex;
+}
+
+.indexes {
+  width:95%;
+  margin-top: 15px;
+  float: left;
+  margin-left: 1rem;
+  margin-right: 30px;
+  border-bottom: 1px solid rgb(228, 228, 228);
+  border-top: 1px solid rgb(228, 228, 228);
+}
+
+.content-indexes {
+  margin-top: 20px;
+}
+
+.grid-index {
+  display: grid;
+  grid-template-columns: auto auto auto auto auto auto;
+}
+
+.grid-data {
+  margin-right: 2rem;
+}
+
+.grid-data li {
+  padding-bottom: 5px;
+}
+
+.grid-data:last-child {
+  border-left: 1px solid rgb(228,228,228);
+}
+
+.site-info {
+  width:1200px;
+  margin-top: 10rem;
+  margin-left: 150px;
+
+}
+
+.site-info nav a {
+  padding-right: 15px;
+  font-size: 75%;
+  text-align: center;
+}
+
+/* footer end */
+
+.grid-container {
+  display: grid;
+  margin-left: 2rem;
```

Comment:
Try setting the margin to the left and right to `auto`, for example: `margin: 0 auto`. That might solve the issue with the misalignment on wider screens. Take a look at this: https://www.w3schools.com/css/css_align.asp